CONTRIBUTING.md

```diff
@@ -86,37 +86,62 @@ They are written in [Cypress](https://www.cypress.io/).
 
 Visual regression tests are run for all "All Combinations" stories, or you can add specs individually for other stories.
 
-You can run UI tests as the CI would with:
+Docker (and Compose) is used for running E2E tests, so first install it via Homebrew:
+
+    brew install --cask docker
+
+Then (only once) build the local container:
+
+    docker-compose build
+
+Now you can run UI tests with:
 
     yarn e2e
 
-That runs all specs and generates coverage reports.
+This runs all specs and generates coverage reports.
+
+For writing tests locally, you would first need to generate snapshots so they are match your environment:
+
+    CYPRESS_updateSnapshots=true yarn concurrently -k --success first -n ,cypress:run yarn:e2e:serve "yarn wait-on http://localhost:6006 && yarn cypress run -c video=true"
 
-Beware that environment differences between CI and your local machine might be enough for a diff to be detected.
+Then open Cypress with:
 
-For writing tests locally:
+    yarn concurrently -k --success first -n ,cypress:open yarn:e2e:serve "yarn cypress open --env failOnSnapshotDiff=false"
```

Comment:
Let's make this a script like you suggested before, it's a bit verbose otherwise.